dimod/quadratic/quadratic_model.py

```diff
@@ -597,6 +629,7 @@ def relabel_variables(self, mapping: Mapping[Variable, Variable],
 
     def relabel_variables_as_integers(self, inplace: bool = True
                                       ) -> Tuple['QuadraticModel', Mapping[Variable, Variable]]:
+        """Relabel the variables as `[0, n)` returning the mapping."""
```

Comment:
"returning the mapping" --> "and return the mapping" if you want the return value described in the one-liner